helpers/constants.js

```diff
@@ -37,6 +43,7 @@ export const SUPPORTED_VERSIONS = {
   UNINSTALLED_EXTENSIONS_API: '1.2.0', // Version that supports uninstalled extensions on Discovery API
   PLATFORM: '1.0.0', // Version that supports platform
   PLUGIN_SERVICES: '1.2.0', // Version that supports plugin services TODO: up this after development
+  SUGGESTED_FEATURES: '1.2.0', // Version that supports suggested features
```

Comment:
```suggestion
  SUGGESTED_FEATURES: '1.7.0', // Version that supports suggested features
```
(assuming that we release this for next minor FCL release)